packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.service.js

```diff
@@ -440,27 +444,29 @@ export default class PciProjectStorageBlockService {
     return this.$q.when(consumptionVolumeAddons);
   }
 
-  getSnapshotPriceEstimation(projectId, storage) {
-    return this.CucPriceHelper.getPrices(projectId).then((catalog) => {
-      const price = get(
-        catalog,
-        `${VOLUME_SNAPSHOT_CONSUMPTION}.${storage.region}`,
-        get(catalog, VOLUME_SNAPSHOT_CONSUMPTION, false),
-      );
-      if (price) {
-        const snapshotPrice =
-          (price.priceInUcents * moment.duration(1, 'months').asHours()) /
-          100000000;
-        return {
-          price: snapshotPrice,
-          priceText: price.price.text.replace(
-            /\d+(?:[.,]\d+)?/,
-            round(snapshotPrice.toString(), 2),
-          ),
-        };
-      }
-      return Promise.reject();
-    });
+  getSnapshotPriceEstimation(projectId, storage, catalogEndpoint) {
+    return this.CucPriceHelper.getPrices(projectId, catalogEndpoint).then(
+      (catalog) => {
+        const price = get(
+          catalog,
+          `${VOLUME_SNAPSHOT_CONSUMPTION}.${storage.region}`,
+          get(catalog, VOLUME_SNAPSHOT_CONSUMPTION, false),
+        );
+        if (price) {
+          const snapshotPrice =
+            (price.priceInUcents * moment.duration(1, 'months').asHours()) /
```

Comment:
Nitpick: can you move this code to separate function to improve the readability like formatPrice()?

Reply:
Not an issue, can stay. Probably in another task.
We have to start the QC.